parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -202,7 +202,7 @@ Expand-Archive scripts.zip -DestinationPath "C:\\AzureData\\"
 $useContainerD = ($global:ContainerRuntime -eq "containerd")
 $global:KubeClusterConfigPath = "c:\k\kubeclusterconfig.json"
 $fipsEnabled = [System.Convert]::ToBoolean("{{ FIPSEnabled }}")
-$windowsSecureTlsEnabled = "{{GetVariable "windowsSecureTlsEnabled" }}"
+$windowsSecureTlsEnabled = [System.Convert]::ToBoolean("{{GetVariable "windowsSecureTlsEnabled" }}");

```

Comment:
nit: why end with `;` when no `;` around this line

Reply:
Both work. I copied it from line 171. You can see above lines ending with ";", for example, line 157 - 177. We can normalize them later.